GRDB/QueryInterface/Schema/TableDefinition.swift

```diff
@@ -619,6 +619,19 @@ public final class ColumnDefinition {
         var deferred: Bool
     }
     
+    /// The GeneratedColumnQualification enum defines whether a generated column is virtual or stored
+    ///
+    /// See https://sqlite.org/gencol.html#virtual_versus_stored_columns
+    public enum GeneratedColumnQualification {
+        case virtual
+        case stored
+    }
+    
+    private struct GeneratedColumnConstraint {
+        var expression: SQLExpression
```

Comment:
Thank you for using `SQLExpression` here. The type fits the SQL grammar. And it will be easy to add support for expressions that are not raw SQL. 👍